scoreCard.js

```diff
@@ -0,0 +1,14 @@
+const score = (user) => {
+    let scores = user.map((array, i) => {
+        if ((array[0] === 10) && (i < 9)) {
+          return 10 + user[i+1].reduce((a, b) => a + b);
```

Comment:
I like the use of the reduce method. Makes the function a lot shorter with less methods.